xbmc/settings/AdvancedSettings.h

```diff
@@ -116,6 +121,9 @@ class CAdvancedSettings : public ISettingCallback, public ISettingsHandler
     void AddSettingsFile(const std::string &filename);
     bool Load(const CProfileManager &profileManager);
 
+    int RegisterSettingsLoadedCallback(AdvancedSettingsCallback callback);
+    void UnregisterSettingsLoadedCallback(int);
```

Comment:
Have you intentionally not provided a name for the parameter?

Reply:
no, I missed it. Added doxygen while I was at it.